YourToDo/templates/planner/planner.html

```diff
@@ -8,180 +8,171 @@
         <!-- Table containing top content (could potentially be used for all views) -->
         <table class="planner-container">
             <!-- Column Specification: Left sidebar, 7 days -->
-            <col class="planner-week-left-sidebar">
-            <col class="planner-week-days-day">
-            <col class="planner-week-days-day">
-            <col class="planner-week-days-day">
-            <col class="planner-week-days-day">
-            <col class="planner-week-days-day">
-            <col class="planner-week-days-day">
-            <col class="planner-week-days-day">
+            <col class="planner-month-days-day">
+            <col class="planner-month-days-day">
+            <col class="planner-month-days-day">
+            <col class="planner-month-days-day">
+            <col class="planner-month-days-day">
+            <col class="planner-month-days-day">
+            <col class="planner-month-days-day">
+            <col class="planner-month-right-sidebar">
```

Comment:
Possibly make this column a little narrower, at least on large displays. It seemed a bit too bit & we could use extra space for the 7 day columns.